### PR TITLE
Added detail about open ports in Umbraco Cloud

### PR DESCRIPTION
**Pull request by netcamo**
## Description

_What did you add/update/change?_
I have added detail about open ports 

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_
do not merge this in until I write back when it can be merged, as am waiting for an approval internally

### CODE REVIEW

**Comment by netcamo** [comment]
Waiting for review and approval

**Review by pijemcolu — APPROVED** [review]

File: umbraco-cloud/security.md
Comment (alina-tincas) [inline umbraco-cloud/security.md:101]:
```suggestion
Some scanning tools will report some other ports open due to Cloudflare's default support on those ports. However, all traffic on those ports is managed by Umbraco Cloud and never reaches your project. You can read more about the Cloudflare Network Ports in the [Cloudflare Documentation](https://developers.cloudflare.com/fundamentals/reference/network-ports/).
```

**Review by alina-tincas — APPROVED** [review]

File: umbraco-cloud/security.md
Comment (alina-tincas) [inline umbraco-cloud/security.md:104]:
```suggestion
{% hint style="info" %}
Some scanning tools will report some other ports open due to Cloudflare's default support on those ports. However, all traffic on those ports is managed by Umbraco Cloud and never reaches your project. You can read more about the Cloudflare Network Ports in the [Cloudflare Documentation](https://developers.cloudflare.com/fundamentals/reference/network-ports/).
{% endhint %}

```

**Pull request merged**
